server/src/definition/DbtDefinitionProvider.ts

```diff
@@ -31,10 +31,11 @@ export class DbtDefinitionProvider {
     position: Position,
     jinjaType: JinjaType,
   ): DefinitionLink[] | undefined {
-    const modelDefinitions =
-      packageName && jinjaType === JinjaType.EXPRESSION
-        ? this.modelDefinitionProvider.provideDefinitions(document, position, jinja, packageName)
-        : undefined;
+    if (jinjaType === JinjaType.UNKNOWN) {
+      return undefined;
+    }
+
+    const modelDefinitions = packageName ? this.modelDefinitionProvider.provideDefinitions(document, position, jinja) : undefined;
```

Comment:
Could you remind what does it mean if `packageName === undefined` here?

Reply:
Now `packageName` is not needed here.